testsuite/pnm/ref/out.txt

```diff
@@ -46,6 +51,7 @@ src/rgb-binary.ppm   :   32 x   32, 3 channel, uint8 pnm
     channel list: R, G, B
     oiio:ColorSpace: "Rec709"
     pnm:binary: 0
+    pnm:pfmflip: 1
```

Comment:
I don't think we should add the pnm:pfmflip attribute reporting to files we read that are not pfm files.

Reply:
fixed